content/ja/docs/concepts/configuration/secret.md

```diff
@@ -752,6 +626,34 @@ echo $SECRET_PASSWORD
 ```
 1f2d1e2e67df
 ```
+// diffの762あたり
+
+
+## Immutable Secrets {#secret-immutable}
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+Kubernetesベータ機能_ImmutableSecrets and ConfigMaps_は、個々のSecretsとConfigMapsをimutableとして設定するオプションを提供します。Secret（少なくとも数万の、SecretからPodへの一意のマウント）を広範囲に使用するクラスターの場合、データの変更を防ぐことには次の利点があります。
```

Comment:
Current code displays following:

---
![screenshot-localhost_1313-2021 05 18-13_35_10](https://user-images.githubusercontent.com/748740/118591837-dab7ca00-b7df-11eb-8f7f-eeb27b254a4d.png)
---

`_ImmutableSecrets and ConfigMaps_` is displayed as is.

We can use `*<word>*` instead of `_<word>_` if the `<word>` has not space on both edge.
```suggestion
Kubernetesベータ機能*ImmutableSecrets and ConfigMaps*は、個々のSecretsとConfigMapsをimutableとして設定するオプションを提供します。Secret（少なくとも数万の、SecretからPodへの一意のマウント）を広範囲に使用するクラスターの場合、データの変更を防ぐことには次の利点があります。
```